src/platform/packages/shared/kbn-unified-tabs/src/components/tab/tab.test.tsx

```diff
@@ -45,7 +45,9 @@ describe('Tab', () => {
       />
     );
 
-    expect(screen.getByText(tabItem.label)).toBeInTheDocument();
+    const tabButton = screen.getByTestId(tabButtonTestSubj);
+    expect(tabButton).toBeInTheDocument();
+    expect(tabButton).toHaveTextContent(tabItem.label);
```

Comment:
Since TabPreview is hidden via opacity (to use some animation when it's appearing and hiding) and it uses label tests were failing. Switched selectors to be more specific